internal/mempool/mempool.go

```diff
@@ -2159,4 +2194,456 @@ func New(cfg *Config) *TxPool {
 		staged:          make(map[chainhash.Hash]*dcrutil.Tx),
 		stagedOutpoints: make(map[wire.OutPoint]*dcrutil.Tx),
 	}
+
+	mp.miningView = &txMiningView{
+		rejected:      make(map[chainhash.Hash]struct{}),
+		txGraph:       mp.newTxDescGraph(),
+		txDescs:       []*mining.TxDesc{},
```

Comment:
This should be `nil` to avoid needlessly allocating an empty array this is just going to have to replaced when it gets appended to anyway.  It is perfectly safe to append to a `nil` array, unlike maps.